Model/JWTRequest.php

```diff
@@ -72,7 +72,12 @@ private function buildAuthHeader($method, $restUrl)
 
     private function buildURL($restUrl)
     {
-        return $this->tenant->getBaseUrl().$restUrl;
+        // Jira return absolute self links, so its more easy to work with get with absolute urls in such cases  
```

Comment:
@thecatontheflat 
well, not exactly
code like
$jwtRequest = new JWTRequest($this->getUser());
$watchesClass = $jwtRequest->get($absoluteUrl);
will not work, because $jwtRequest->get will call build url